src/content/en/2020/page-weight.md

```diff
@@ -23,9 +23,9 @@ featured_stat_label_3: The median number of mobile page requests
 
 Page weight is one of the simpler metrics available. Much like stepping on a human scale to get a sense of your personal weight (well, mass really, but you get it), loading a page will provide a sense of the number and size of resources collected and requested. But as the web and web pages have matured and grown, so have associated metrics — such as page weight. It can affect a page's performance much like personal weight (mass) can do the same. This chapter will take a deeper dive and peel back the layers of web pages and see what it is that constitutes a page's weight at the possible detriment of the end user: you, I, us.
 
-## #PageWeightStillMatters
+### PageWeightStillMatters
```

Comment:
I can see why you thought this was a mistake but it's actually not. This is the author using a cool, twitter-style hash tag in his text. It should still be a level-2 heading and not a level 3.

```suggestion
### PageWeightStillMatters
```
```suggestion
## #PageWeightStillMatters
```